src/OpenTK/Platform/Android/AndroidDisplayDeviceDriver.cs

```diff
@@ -12,25 +12,37 @@ namespace OpenTK.Platform.Android
     internal class AndroidDisplayDeviceDriver : IDisplayDeviceDriver
```

Comment:
Alright, just as long as it doesn't straight up doesn't work on Android :P Returning some placeholder values is fine for now.